coverage-tool/diff_coverage.py

```diff
@@ -70,22 +77,21 @@ def has_changes(field):
                 if value[OLD_KEYWORD] == value[NEW_KEYWORD]:
                     print("| " * indent + f"{key}: {value[OLD_KEYWORD]}{percent}")
                 else:
-                    if not self.no_color:
-                        if value[OLD_KEYWORD] == 0:
-                            color = "green"
-                        elif value[NEW_KEYWORD] == 0:
-                            color = "red"
-                        elif value[OLD_KEYWORD] > value[NEW_KEYWORD]:
-                            color = "yellow" if key != TOTAL_KEYWORD else "red"
-                        else:
-                            color = "blue" if key != TOTAL_KEYWORD else "green"
+                    if value[OLD_KEYWORD] == 0:
+                        color = "green"
+                    elif value[NEW_KEYWORD] == 0:
+                        color = "red"
+                    elif value[OLD_KEYWORD] > value[NEW_KEYWORD]:
+                        color = "yellow" if key != TOTAL_KEYWORD else "red"
+                    else:
+                        color = "blue" if key != TOTAL_KEYWORD else "green"
```

Comment:
what's blue for?

Reply:
The number of files for a field increasing, but not from zero (in which case it's green).

Blue and yellow changes don't affect the total coverage percentage, and the colors are just for distinguishing between increases and decreases. I can remove them in the emoji output mode because they can be quite distracting. WDYT?

Reply:
If it's going to be described somewhere in the output what they mean, like a footnote for example, I'm OK with leaving them in.

Reply:
lgtm